python/matcher_test.go

```diff
@@ -39,7 +40,7 @@ func (tc matcherTestcase) Run(t *testing.T) {
 
 // TestMatcher tests the python matcher.
 func TestMatcher(t *testing.T) {
-	tt := []matcherTestcase{
+	testcases := []matcherTestcase{
```

Comment:
can you add a test case for a package version less than the "introduced" version?